distribution/src/config/jvm.options

```diff
@@ -75,3 +75,8 @@ ${error.file}
 
 # JDK 9+ GC logging
 9-:-Xlog:gc*,gc+age=trace,safepoint:file=${loggc}:utctime,pid,tags:filecount=32,filesize=64m
+
+# due to internationalization enhancements in JDK 9 Elasticsearch need to set the provider to COMPAT otherwise
+# time/date parsing will break in an incompatible way for some date patterns and locals
+8-:-Djava.locale.providers=SPI,JRE
```

Comment:
Shouldn't this just be `8`, not `8-`?

Reply:
true, this should just be `8` - removed in favour of conditional check in `SystemJvmOptions`